FFXIVClientStructs/FFXIV/Client/UI/RaptureAtkModule.cs

```diff
@@ -137,6 +137,9 @@ public unsafe partial struct RaptureAtkModule {
     [VirtualFunction(63), GenerateStringOverloads]
     public partial bool OpenMapWithMapLink(CStringPointer mapLink);
 
+    [MemberFunction("48 89 5C 24 ?? 48 89 74 24 ?? 48 89 7C 24 ?? 55 41 56 41 57 48 8B EC 48 83 EC 40 4C 8B F1")]
+    public partial void HandleItemMove(void* outValue, AtkValue* atkValues);
```

Comment:
Please use the arguments and return type of [`AtkModule.CallbackHandlerDelegate`](https://github.com/aers/FFXIVClientStructs/blob/main/FFXIVClientStructs/FFXIV/Component/GUI/AtkModule.cs#L18):

```suggestion
    public partial AtkValue* HandleItemMove(AtkValue* returnValue, AtkValue* values, uint valueCount);
```

Reply:
Thank you! Was unaware of this